### PR TITLE
Fsf go

### PR DESCRIPTION
**Pull request by igorljubuncic**
Added build-packages, gcc for httplab.

### CODE REVIEW

**Comment by evandandrea** [comment]
I can confirm that the snapcraft.yaml change builds and runs correctly.

**Comment by evandandrea** [comment]
I've committed the snapcraft.yaml changes to snapcraft-docs/httplab

**Comment by therealjuan** [comment]
@igorljubuncic can you please rebase from master if this is ready to merge?

**Comment by evandandrea** [comment]
@igorljubuncic you've accidentally deleted _fsf_go and _fsf_java:
https://github.com/canonical-websites/snapcraft.io/pull/1571/files

**Comment by igorljubuncic** [comment]
> @igorljubuncic you've accidentally deleted _fsf_c and _fsf_java:
> https://github.com/canonical-websites/snapcraft.io/pull/1571/files

I've pushed a new change, should be ok now.

File: templates/home/_fsf_go.html [hunk 1]
```diff
@@ -30,7 +30,9 @@ <h4>Here’s how httplab uses it:</h4>
     <b>go-importpath</b>: github.com/gchaincl/httplab
     <b>source</b>: .
     <b>source-type</b>: git
-
+  <b> build-packages</b>:
```
Comment (evandandrea) [inline templates/home/_fsf_go.html]: Indentation is off:
screenshot 2019-02-11 at 15 13 57

**Comment by igorljubuncic** [comment]
Made changes as requested.

**Pull request merged**